README.md

```diff
@@ -11,7 +11,7 @@ URLs are crawled via _aiohttp_, catalog and crawled metadata are stored in a _Po
 ### Create database structure
 
 Install udata-hydra dependencies and cli.
-`make deps`
+`poetry install`
```

Comment:
Should we add a link to poetry, for installation instruction?

Reply:
This would mean adding a link to poetry installation instructions for all our projects, do we want to? Maybe only on udata since no one should reasonably be using this without it?